apps/volunteer/shifts/talks.py

```diff
@@ -0,0 +1,83 @@
+from ..event_date import edt
+
+herald_pattern = [
+            {
+                "first": edt("fri", "09:00:00"),
+                "final": edt("fri", "21:00:00"),
+                "min": 1,
+                "max": 1,
+            } for d in ["fri", "sat", "sun"]
+        ]
+stage_pattern = [
+    {"first": edt(d, "10:00:00"),
+     "final": edt(d, "18:00:00"),
+     "min": 1,
+     "max": 1,
+    } for d in ["fri", "sat", "sun"]
+]
+vm_pattern = [
+    {"first": edt(d, "10:00:00"),
+     "final": edt(d, "18:00:00"),
+     "min": 1,
+     "max": 1,
+     "base_duration": 240
+    } for d in ["fri", "sat", "sun"]
+]
+
+talks_shifts = {
+    "Green Room Runner": {
```

Comment:
We should confirm this, main stage content ends 8pm most days and then switches to performances.

Reply:
Similarly for the heralds.